components/country/overview.js

```diff
@@ -139,6 +141,7 @@ const Overview = ({
         <FormattedMessage id='Country.Overview.Heading.TestsByClass' />
       </Heading>
       <TestsByGroup
+        notEnoughData={measurementCount < LOW_DATA_THRESHOLD}
```

Comment:
@hellais Please review if this is a good measure to detect when to show 'No Data Available' message across the overview chart. For reference, the `/country/IO` page has 4 measurements here, but renders an empty chart.

Reply:
What you really want to be looking at is if there is data from the last 30 days.

Reply:
You should not be looking at the absolute measurementCount, but rather the `testCoverage` and `networkCoverage` arrays.

A future optimisation could be to have an API endpoint that is just for this (or maybe do it as part of other API endpoints).

In any case it's not enough to look at the absolute `measurementCount`, because if a country has many measurements, but they are all in the past we still don't have enough data to plot the overview chart.

Reply:
We may also want to change the copy to "No data available from the last 60 days" cc @agrabeli

Reply:
:+1: done